cmd/kube-apiserver/app/server.go

```diff
@@ -96,6 +97,7 @@ func NewAPIServer() *APIServer {
 		EventTTL:               1 * time.Hour,
 		AuthorizationMode:      "AlwaysAllow",
 		AdmissionControl:       "AlwaysAdmit",
+		EtcdPathPrefix:         tools.DefaultEtcdPathPrefix,
```

Comment:
Please remove DefaultEtcdPathPrefix from the tools package, and put it in the master package.